client/src/hooks/useFriends.js

```diff
@@ -0,0 +1,12 @@
+import { useEffect, useState } from "react";
+import { getItemsFromLocalStorage } from "../utils/itemStorage";
+
+export default function useFriends() {
+  const [friends, setFriends] = useState([]);
+
+  useEffect(() => {
+    const friends = getItemsFromLocalStorage("friends");
+    setFriends(friends);
+  }, []);
+  return [friends, setFriends];
```

Comment:
here, you can just leave out the setFriends. or you can not request it in the file where you are using the custom hook